### PR TITLE
studio: add main menu

### PR DESCRIPTION
**Pull request by achen5671**
## References

https://tsu.atlassian.net/browse/IR-2524

### CODE REVIEW

File: packages/editor/src/components/toolbar/Toolbar2.tsx [hunk 1]
```diff
@@ -114,23 +114,87 @@ export default function Toolbar() {
...
-          }}
-        />
-        {/* TO BE ADDED
-        <div className="flex items-center gap-2.5 rounded-full bg-theme-surface-main p-0.5">
+        <div className="flex items-center">
+          <Button
+            endIcon={<MdOutlineKeyboardArrowDown size="1.5em" className="text-[#A3A3A3]" />}
+            rounded="none"
+            startIcon={
+              <svg xmlns="http://www.w3.org/2000/svg" width="19" height="21" viewBox="0 0 19 21" fill="none">
```
Comment (HexaField) [inline packages/editor/src/components/toolbar/Toolbar2.tsx]: please store svgs separately and not inline

**Review by HexaField — DISMISSED** [review]

File: packages/editor/src/components/toolbar/Toolbar2.tsx [hunk 2]
```diff
@@ -113,30 +113,61 @@ export default function Toolbar() {
...
+            className="-ml-1 border-0 bg-transparent p-0"
+            onClick={(event) => {
+              // anchorOpen.set(true)
+              // anchorPosition.set({ left: event.clientX - 5, top: event.clientY - 2 })
+              // anchorEl.set(event.currentTarget)
+            }}
+          /> */}
+        </div>
+        {/* TO BE ADDED */}
+        {/* <div className="flex items-center gap-2.5 rounded-full bg-theme-surface-main p-0.5">
```
Comment (aditya-mitra) [inline packages/editor/src/components/toolbar/Toolbar2.tsx]: We should remove these lines. This results in code bloat.
To add this line later, we can put it in a separate commit (and then overwrite without those lines) and cherry-pick it when the requirements comes in.

File: packages/ui/src/components/editor/layout/ContextMenu.tsx [hunk 3]
```diff
@@ -93,7 +93,7 @@ export const ContextMenu = ({
         {open && anchorEl && (
           <div
             ref={menuRef}
-            className="absolute z-[200] w-40 rounded-lg bg-neutral-900 shadow-lg"
+            className="absolute z-[200] h-[228px] w-[180px] rounded-lg bg-neutral-900 shadow-lg"
```
Comment (aditya-mitra) [inline packages/ui/src/components/editor/layout/ContextMenu.tsx]: let's use dynamic context menu height and tailwind value for width

**Review by aditya-mitra — CHANGES_REQUESTED** [review]
The hamburger menu looks different from [the figma designs](https://www.figma.com/design/DXwFHGa5g0peqqSjyjpxQz/Studio-Design?node-id=266-20611&t=ob9ykvuv9CY1jCPY-0).

current
image


figma
image

**Comment by achen5671** [comment]
> The hamburger menu looks different from [the figma designs](https://www.figma.com/design/DXwFHGa5g0peqqSjyjpxQz/Studio-Design?node-id=266-20611&t=ob9ykvuv9CY1jCPY-0).
> 
> current image
> 
> figma image

we got new designs for the navbar here and we decided to remove the panel menu for now as it's not used. let me know if you want the figma file :)

image

File: packages/editor/src/components/toolbar/Toolbar2.tsx (hunk 2 above)
Comment (achen5671) [reply packages/editor/src/components/toolbar/Toolbar2.tsx]: sounds good!

File: packages/editor/src/components/toolbar/Toolbar2.tsx [hunk 4]
```diff
@@ -83,7 +83,7 @@ const generateToolbarMenu = () => {
     },
     {
       name: t('editor:menubar.saveScene'),
-      hotkey: `${cmdOrCtrlString}+s`,
+      // hotkey: `${cmdOrCtrlString}+s`,
```
Comment (aditya-mitra) [inline packages/editor/src/components/toolbar/Toolbar2.tsx]: why did we comment this out?
have not seen the design yet but if the hotkey is absent in designs, let's remove the `hotkey` key value

**Review by aditya-mitra — CHANGES_REQUESTED** [review]

**Review by aditya-mitra — APPROVED** [review]

**Pull request merged**